src/terminal-screen.h

```diff
@@ -35,7 +35,7 @@ typedef enum
     FLAVOR_VOIP_CALL,
     FLAVOR_EMAIL,
     FLAVOR_SKEY
-} TerminalURLFlavour;
+} TerminalURLFlavor;
```

Comment:
If you're renaming this enum to make it consistent with its values, can you do it in a separate commit?

Reply:
I mean the first: move renaming `TerminalURLFlavor` to a separate commit.

Reply:
BTW all renamings are in separate commit 285a0fcc8f29a7cc5ccb9d8f7796c9f2f68ff961 already
I've just mentioned TerminalURLFlavor renaming in commit message.

Reply:
Ah, right, my bad, I just saw the refactoring commit 👍